library/java/io/envoyproxy/envoymobile/Envoy.java

```diff
@@ -11,7 +11,7 @@
 
   public void load() { System.loadLibrary("envoy_jni"); }
 
-  public void run(Context context, String config) {
+  public void run(final Context context, final String config) {
```

Comment:
I got a complaint on the build without this change but I think we can just ignore it since this file is intended to change completely